src/olympia/editors/models.py

```diff
@@ -78,7 +78,7 @@ def new_editors():
 
         return [dict(user=i.arguments[1],
                      created=i.created)
-                for i in items]
+                for i in items if i.arguments[1] in group.users.all()]
```

Comment:
The drawback is that the final list may contain less than five entries.

The problem is that `arguments` is a JSON string like `u'[{"access.group": 50002}, {"users.userprofile": 5497308}]'`. I want to check whether `5497308` is in `group.users.all()`.
Ideally, I'd like to do something like `.filter(_arguments[1]__in=group.users.all())` but of course that doesn't work.
Is there a way to do that?